R/utilities-pk-analyses.R

```diff
@@ -27,3 +27,20 @@ calculatePKAnalyses <- function(results, simulation) {
 toPKParameter <- function(netPKParameters) {
   toObjectType(netPKParameters, PKParameter)
 }
+
+
+#' Saves the pK-analyses  to csv file
+#'
+#' @param pkAnalyses pK-Analyses to exporte (typically calculated using \code{calculatePKAnalyses} or imported from file)
+#' @param simulation Instance of a simulation used to calculate the pK-Analyses
+#' @param filename Full path of where the pK-Analyses will be saved.

```

Comment:
Please rename `filename` to `outputFilename` or something, so it is clear _what_ the file is for.

Reply:
`outputCSVname`...

Is it really just the name, or the full path to the output file?

Reply:
I like when I know how to use the function without reading the comments :)

Reply:
Of course. but how do you know what parameters to call, the order etc.... At some point you need to read the docs or at least look at the intellisense in RStudio (which shows the description of parameters)

Reply:
Hm, I'm not getting the description in the intellisense, only the names of the parameters... Have to check the settings.

Reply:
I think it only works when you import the packages (not in dev). 
At any rate, I'll rename. But we have to assume that people are going to read the F.. manual otherwise this will be a lost cause ;)

Reply:
done. filePath